core/audits/redirects.js

```diff
@@ -96,10 +98,10 @@ class Redirects extends Audit {
     const metricResult = await LanternInteractive.request(metricComputationData, context);
 
     /** @type {Map<string, LH.Gatherer.Simulation.NodeTiming>} */
-    const nodeTimingsByUrl = new Map();
```

Comment:
similar to the `navigationId` change above, this can get confused if there are multiple resources per URL. The graph has the full records, so this can match on the actual unique ID.